cfme/fixtures/v2v_fixtures.py

```diff
@@ -140,35 +147,98 @@ def create_tags(appliance, transformation_method):
     return tag1, tag2
 
 
-def set_conversion_instance_for_rhev(appliance, transformation_method, rhev_hosts):
+def get_vddk_url():
+    """Get vddk url from cfme_data"""
+    vddk_version = "v6_5_2_v2v"
```

Comment:
Does it need to be hard coded in here?

Reply:
We should not look it as a hard-coding, `v6_5_2_v2v` is the just a key to retrieve url from cred.yaml. We are using VDDK version 6.5.2 here but version doesn't matter since we can use any 6.5+ VDDK with 6/6.5/6.7 versions.